scripts/Install-VSCode.ps1

```diff
@@ -122,6 +126,10 @@ param(
     [ValidateSet("stable","insider")]
     [string]$BuildEdition = "stable",
 
+    [Parameter()]
+        [switch]
```

Comment:
The convention in the rest of the file seems to be putting the type constraint on the same line as the parameter

Reply:
fixed and about to check in